pages/advanced.tsx

```diff
@@ -1,32 +1,64 @@
 import React, { useContext, useState } from 'react';
 
-import { Button, Layout, Sidebar, GenerateConfigModal } from '../components';
+import { Layout, Sidebar } from '../components';
+import ProbeCard from '../components/probe-card';
+import { ProbeContext } from '../contexts/probe-context';
+import { Probe } from '@hyperjumptech/monika/lib/interfaces/probe';
+import { Notification } from '@hyperjumptech/monika/lib/interfaces/notification';
+import { Button, GenerateConfigModal } from '../components';
 import NotifCard from '../components/notif-card';
 import { NotificationContext } from '../contexts/notification-context';
 
 export default function Advanced(): JSX.Element {
   const { notificationData, handleAddNotification } = useContext(
     NotificationContext
   );
+  const { probeData, handleAddProbe } = useContext(ProbeContext);
   const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
 
-  const renderProbe = () => {
-    return <div>Hello Probe</div>;
+  type Menu = 'Probe' | 'Notification';
+  const [activeMenu, setActiveMenu] = useState<Menu>('Probe');
```

Comment:
Why we need to move `activeMenu` from `sidebar` to this component?

Reply:
Could you please check again as I have updated the sidebar component to be more reusable :grinning: